mailpoet/tests/performance/tests/newsletter-searching.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable import/no-default-export */
+/**
+ * External dependencies
+ */
+import { sleep, check, group } from 'k6';
+import { chromium } from 'k6/experimental/browser';
+import { randomIntBetween } from 'https://jslib.k6.io/k6-utils/1.1.0/index.js';
+
+/**
+ * Internal dependencies
+ */
+import {
+  baseURL,
+  thinkTimeMin,
+  thinkTimeMax,
+  headlessSet,
+  timeoutSet,
+} from '../config.js';
+import { authenticate } from '../utils/helpers.js';
+/* global Promise */
+
+export function newsletterSearching() {
+  const browser = chromium.launch({
+    headless: headlessSet,
+    timeout: timeoutSet,
+  });
+  const page = browser.newPage();
+
+  group('Emails - Search for a newsletter', () => {
+    page
+      .goto(`${baseURL}/wp-admin/admin.php?page=mailpoet-newsletters`, {
+        waitUntil: 'networkidle',
+      })
+
+      .then(() => {
+        authenticate(page);
+      })
+
+      .then(() => {
+        return Promise.all([
+          page.waitForNavigation({ waitUntil: 'networkidle' }),
+        ]);
+      })
```

Comment:
```suggestion
      .then(() => page.waitForNavigation({ waitUntil: 'networkidle' }))
```
I am not familiar with k6 but would it work to remove the Promise.all as there is only one?

Reply:
If I leave it without `Promise.all` the test will brake at that step so the only solution I have currently working 100% is to leave inside `Promise.all`. 
However, I am working on a different way to authenticate which will be without logging to WP Admin, so this will be removed along with `authenticate` method.